.prettierignore

```diff
@@ -14,3 +14,10 @@ src/api/status/public/assets
 src/web/public/sw*
 src/web/public/workbox*
 src/api/status/src/views
+
+# Ignore these directories for the docusaurus docs website
+src/docs/.docusaurus
+src/docs/build
+src/docs/node_modules/
+src/docs/package.json
+src/docs/package-lock.json
```

Comment:
I think these might be unnecessary, since we already have `package.json` and `package-lock.json` in the list, and the [`.gitignore`](https://git-scm.com/docs/gitignore#_examples) rules are the same for .prettierignore.

This is more of a nitpick, so it might not even be necessary to change, just letting you know.